sparrow-py/pysrc/sparrow_py/_expr.py

```diff
@@ -288,8 +290,9 @@ def is_not_null(self) -> "Expr":
         return Expr.call("is_valid", self)
 
     def collect(
-        self, max: Optional[int] = None, window: Optional[Window] = None
+        self, max: Optional[int], window: Optional[Window] = None
```

Comment:
could've sworn I had this. Maybe I stashed a change I meant to add

Reply:
🤷 it's fine